config/export/submissions/type-export.js

```diff
@@ -0,0 +1,587 @@
+    lblodBesluit: "http://lblod.data.gift/vocabularies/besluit/",
+    foaf: "http://xmlns.com/foaf/0.1/",
+    org: "http://www.w3.org/ns/org#",
+    skos: "http://www.w3.org/2004/02/skos/core#",
+    rdf: "http://www.w3.org/1999/02/22-rdf-syntax-ns#",
+    rdfs: "http://www.w3.org/2000/01/rdf-schema#",
+    ext: "http://mu.semte.ch/vocabularies/ext/",
+    regorg: "https://www.w3.org/ns/regorg#",
+    person: "http://www.w3.org/ns/person#",
+    schema: "http://schema.org/",
+    prov: "http://www.w3.org/ns/prov#",
+    adms: "http://www.w3.org/ns/adms#",
+    mu: "http://mu.semte.ch/vocabularies/core/",
+    pav: "http://purl.org/pav/",
+    dct: "http://purl.org/dc/terms/",
+    nmo: "http://www.semanticdesktop.org/ontologies/2007/03/22/nmo#",
+    nfo: "http://www.semanticdesktop.org/ontologies/2007/03/22/nfo#",
+    nie: "http://www.semanticdesktop.org/ontologies/2007/01/19/nie#",
+    dbpedia: "http://dbpedia.org/ontology/",
+    eli: "http://data.europa.eu/eli/ontology#",
+    meb: "http://rdf.myexperiment.org/ontologies/base/",
+    elod: "http://linkedeconomy.org/ontology#",
+    toezicht: "http://mu.semte.ch/vocabularies/ext/supervision/"
+  },
+  types: [
+    {
+      type: "besluit:Bestuurseenheid",
+      requiredProperties: [
+        "mu:uuid"
+      ],
+      optionalProperties: [
+        "skos:prefLabel",
+        "besluit:werkingsgebied",
+        "besluit:classificatie",
+        "ext:inProvincie",
+        "dct:identifier",
+        "skos:altLabel"
+      ],
+      additionalFilter: ""
+    },
+    {
+      type: "besluit:Bestuursorgaan",
+      requiredProperties: [
+        "mu:uuid"
+      ],
+      optionalProperties: [
+        "skos:prefLabel",
+        "mandaat:bindingEinde",
+        "mandaat:bindingStart",
+        "besluit:bestuurt",
+        "besluit:classificatie",
+        "mandaat:isTijdspecialisatieVan"
+      ],
+      additionalFilter: ""
+    },
+    {
+      type: "prov:Location",
+      requiredProperties: [
+        "mu:uuid"
+      ],
+      optionalProperties: [
+        "rdfs:label",
+        "ext:werkingsgebiedNiveau"
+      ],
+      additionalFilter: ""
+    },
+    {
+      type: "foaf:Person",
+      requiredProperties: [
+        "mu:uuid"
+      ],
+      optionalProperties: [
+        "foaf:firstName",
+        "foaf:familyName"
+      ],
+      additionalFilter: "?s a meb:Submission ; ext:lastModifiedBy ?resource ."
+    },
+    {
+      type: "melding:AutomaticSubmissionTask",
+      requiredProperties: [
+        "mu:uuid"
+      ],
+      optionalProperties: [
+        "dct:created",
+        "prov:generated"
+      ],
+      additionalFilter: ""
+    },
+    {
+      type: "toezicht:InzendingVoorToezicht",
+      requiredProperties: [
+        "mu:uuid"
+      ],
+      optionalProperties: [
+
+      ],
+      additionalFilter: `
+?submission
+  dct:source ?resource ;
+  a meb:Submission ;
+  nmo:sentDate ?sentDate .
+FILTER (?sentDate > "2024-05-01T00:00:00.000Z"^^xsd:dateTime)
```

Comment:
These date filters would be replaced by the date filter variable defined at the top of the config:

```suggestion
${DATE_FILTER}
```